explorer/src/Pos/Explorer/Web/Server.hs

```diff
@@ -366,6 +370,32 @@ getAddressSummary cAddr = do
             ATRedeem     -> CRedeemAddress
             ATUnknown {} -> CUnknownAddress
 
+
+getAddressUtxoBulk
+    :: (HasNodeConfiguration, ExplorerMode ctx m)
+    => [CAddress]
+    -> m [CUtxo]
+getAddressUtxoBulk cAddrs = do
+    when (not explorerExtendedApi) $
+        throwM err405
+        { errReasonPhrase = "Explorer extended API is disabled by configuration!"
+        }
+    addrs <- mapM cAddrToAddr cAddrs
+    utxo <- getFilteredUtxo addrs
+
+    pure $ futxoToCUtxo <$> (M.toList utxo)
```

Comment:
`pure $ futxoToCUtxo <$> (M.toList utxo)` == `pure $ map futxoToCUtxo $ M.toList utxo` == `pure . map futxoToCUtxo . M.toList $ utxo`
Choose any of the latter two, your pick. As "thatguy" (that was his nick, not sure what it is now) mentioned in a discussion we had, using `map` makes it more readable and explicit that we are referring to a list, not some arbitrary `Applicative`.